packages/ui/src/components/Typography/style.less

```diff
@@ -0,0 +1,164 @@
+@import (reference) "@open-condo/ui/src/tokens/variables.less";
+@import (reference) "@open-condo/ui/src/components/style/mixins/typography.less";
+@import "@open-condo/ui/src/components/style/mixins/link.less";
+
+.condo-typography {
+  @condo-typography-decoration-thickness: 1px;
+
+  color: @condo-global-color-black;
+  word-break: break-word;
+
+  &,
+  &-lg {
+    .condo-typography-p1();
+  }
+
+  &-md {
+    .condo-typography-p2();
+  }
+
+  &-sm {
+    .condo-typography-p3();
+  }
+
+  &-secondary {
+    color: @condo-global-color-gray-7;
```

Comment:
Should we change the color naming to a `primary` color? to avoid text hardcode

Reply:
No, we shouldn't. We already discussed it with design team through multiple iterations. What we should do is to create component-based tokens. But for now design team does not have time for this